src/Data/List/Relation/Unary/AllPairs/Properties.agda

```diff
@@ -115,12 +115,16 @@ module _ {R : Rel A ℓ} where
 
 module _ {R : Rel A ℓ} where
 
+  tabulate⁺< : ∀ {n} {f : Fin n → A} → (∀ {i j} → i F.< j → R (f i) (f j)) →
+              AllPairs R (tabulate f)
+  tabulate⁺< {zero}  fᵢ~fⱼ = AllPairs.[]
```

Comment:
I don't think we need the `AllPairs.` qualifiers anywhere in this proof?

Reply:
I removed it.